common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -29,6 +29,11 @@
 public final class CommonUtils {
   private static final Logger LOG = LoggerFactory.getLogger("");
 
+  /**
+   * Get the current time in milliseconds.
+   *
+   * @return current time in milliseconds
+   */
```

Comment:
Yeah for this one its not really a getter, but I think just documenting the `@return` is probably sufficient.
Though I'm not sure why this particular method exists, but that's probably a discussion for another PR.